bindings/Makefile

```diff
@@ -16,12 +17,23 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
 	mkdir -p build/$*
 	java -jar deps/$(CODEGEN_BIN) generate -i $(SWAGGER_SPEC) \
-		-l $* -o build/$*
+		-l $* -o build/$* -c config.json
 	touch build/$*
+	if [ $* = "python" ] ; then \
+		rm -rf $(CLIENT_PATH); \
+		mkdir -p $(CLIENT_PATH)\api; \
+		mkdir -p $(CLIENT_PATH)\models; \
+		cp -rf ./build/python/determined._swagger.client/ $(CLIENT_PATH); \
+		cp -rf ./build/python/determined/_swagger/client/ $(CLIENT_PATH); \
```

Comment:
This doesn't work on my machine, it creates a bunch of strange directories and has some weird behaviors.

I think it is going to have similar effects on your machine, because at a minimum the `$(CLIENT_PATH)\api` will expand to `../harness/determined/_swagger/clientapi` (due to a backslash instead of a forwardslash).

But, as I said over zoom, remove the `-f` flags from `cp` because it does not behave the same on mac as it does on linux.  Then make sure this command is correct by visually inspecting the folders that get created.